src/main.js

```diff
@@ -0,0 +1,27 @@
+const testEnv = 'DATABASE_URL\nADMIN_EMAIL\nDEBUG';
+const alphanumericThatDoesNotStartWithDigit = /^[A-Z_][0-9A-Z_]*$/;
+
+//function that checks if the env value is valid
+//value cannot start with a digit and must only contain alphanumeric characters or an underscore
+function checkValidationOfValues(envSpecString){
+  let envValues = envSpecString.split('\n');//split lines based on \n character
+  envValues = envValues.filter(element =>{
+    if(element.match(alphanumericThatDoesNotStartWithDigit)){//keep valid values
```

Comment:
You can just return `element.match(alphanumericThatDoesNotStartWithDigit)` here. It will do the job 💪.